CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 4.14.0
+
+- Allow use of multiple GA customDimensionIndex.
+- Configurable duration (in days) for AB Test cookie.
+- Add assignment of jQuery explicitly so it's consistent with other modules.
+- Allow base scripts to run within a module loader.
+- Force push tracking branch on new version.
```

Comment:
The audience of the change log is people consuming `govuk_frontend_toolkit`, so we don't need to mention an internal change like the tracking branch.